speech/beta/pom.xml

```diff
@@ -0,0 +1,139 @@
+<!--
+  Copyright 2018, Google Inc.
```

Comment:
Google LLC

Reply:
Only in the newly created beta files, updates to existing files can be left as is.